searx/engines/livespace.py

```diff
@@ -0,0 +1,118 @@
+# SPDX-License-Identifier: AGPL-3.0-or-later
+"""
+ LiveSpace (Videos)
+"""
+
+from urllib.parse import quote_plus
+from json import loads
+from datetime import datetime
+
+# about
+about = {
+    "website": 'https://live.space',
+    "wikidata_id": None,
+    "official_api_documentation": None,
+    "use_official_api": True,
+    "require_api_key": False,
+    "results": 'JSON',
+}
+
+categories = ['videos']
+
+engine_url = 'https://backend.live.space/search/public/stream' + '?page={page}&searchKey={query}&size={size}'
+
+# engine dependent config
+paging = True
+number_of_results = 10
+
+
+def request(query, params):
+    # basic search
+    page = params['pageno'] - 1
+
+    string_args = dict(query=quote_plus(query), page=page, size=number_of_results)
+
+    params['url'] = engine_url.format(**string_args)
+
+    params['headers'] = {'Accept': 'application/json', 'Content-Type': 'application/json'}
+
+    return params
+
+
+def response(resp):
+    results = []
+
+    print(resp.text)
+
+    search_results = loads(resp.text)
+
+    items = search_results.get('result', [])
+
+    for result in items:
+        # for live videos
+        now = datetime.now()
+
+        # stream start
+        startTimeStamp_string = result.get("startTimeStamp", None)
+        startTimeStamp = (
+            None if startTimeStamp_string is None else datetime.utcfromtimestamp(float(startTimeStamp_string) / 1000)
+        )
```

Comment:
```suggestion
        startTimeStamp = None
        if startTimeStamp_string is not None:
          startTimeStamp = datetime.utcfromtimestamp(float(startTimeStamp_string) / 1000)
```
This makes the code more self-explanatory and easier to read